### PR TITLE
Thredds update

### PR DESCRIPTION
**Pull request by jasonb5**
Quick update to fix the paths of output files.

The URL path should be "threddsCWT/public/{user id}/{job number}/{uid}.nc"

This will be used later when we implement protection for user files.

### CODE REVIEW

File: compute/wps/tasks/cdat.py [hunk 1]
```diff
@@ -125,6 +125,8 @@ def base_retrieve(self, attrs, keys, operation, var_name, base_units, output_pat
 
     start = self.get_now()
 
+    os.makedirs(os.path.dirname(output_path))
```
Comment (doutriaux1) [inline compute/wps/tasks/cdat.py]: you might want this in a try/except statement. I think it fails if directory already exists.

File: compute/wps/tasks/job.py [hunk 2]
```diff
@@ -38,15 +38,15 @@ def job_succeeded(self, attrs, variables, output_path, move_path, var_name,
     process = self.load_process(process_id)
 
     if move_path is not None:
+        os.makedirs(os.path.dirname(move_path))
```
Comment (doutriaux1) [inline compute/wps/tasks/job.py]: same here. Fail on existing directory?

**Review by doutriaux1 — APPROVED** [review]

File: compute/wps/tasks/cdat.py (hunk 1 above)
Comment (zshaheen) [reply compute/wps/tasks/cdat.py]: Or you can only make it if doesn't exist.

**Review by zshaheen — APPROVED** [review]

File: compute/wps/tasks/cdat.py (hunk 1 above)
Comment (jasonb5) [reply compute/wps/tasks/cdat.py]: Added try/except, better safe than sorry.

**Pull request merged**